includes/widgets/video.php

```diff
@@ -896,6 +910,7 @@ public function get_embed_params() {
 				'vimeo_title' => 'title',
 				'vimeo_portrait' => 'portrait',
 				'vimeo_byline' => 'byline',
+				'vimeo_background' => 'background',
```

Comment:
Have you tested this? 
Why are you adding a control but not using it's value?
Additionally, keep in mind that this feature is available only if you are a Plus, PRO, Business, or Premium Vimeo member.

Reply:
You are correct, I also realize I have it so the default is to not display the play button which would not be ideal for most people. I guess this was the extent of my expertise, but I think it could be valuable as Vimeo is a very useful service for doing background videos especially with the amount of branding YouTube has recently added to their videos no matter what parameters you add.